packages/fether-react/src/Send/TxForm/TxForm.js

```diff
@@ -105,15 +105,23 @@ class TxForm extends Component {
     return output;
   };
 
-  estimatedTxFee = values => {
+  isEstimatedTxFee = values => {
     if (
-      !values.amount ||
-      !values.gas ||
-      !values.gasPrice ||
-      isNaN(values.amount) ||
-      isNaN(values.gas) ||
-      isNaN(values.gasPrice)
+      values.amount &&
+      values.gas &&
+      values.gasPrice &&
+      !isNaN(values.amount) &&
+      !isNaN(values.gas) &&
```

Comment:
Just seeing this. `values.gas` is a BigNumber, so isNaN(values.gas) is always true?

Reply:
We need `values.gas` to be defined and `!isNaN(values.gas)` to be `true` for it to estimate the gas

When the Account page is loaded:
`values.gas` is `undefined`
`!isNaN(values.gas)` is `false`
`!values.gas.isNaN()` is `undefined`

So it doesn't estimate the gas

After entering just Recipient address or just an Amount:
`values.gas` is `null`
`!isNaN(values.gas)` is `true`
`!values.gas.isNaN()` is `null`

So it doesn't estimate the gas

After entering BOTH a Recipient address and an Amount:
`values.gas` is `BigNumber` (i.e. 21000)
`!isNaN(values.gas)` is `true`
`!values.gas.isNaN()` is `true` 

So it DOES estimate the gas

==============

So it's better to use the `.isNaN()` method provided for BigNumber values